playbooks/install_stack.yaml

```diff
@@ -380,6 +380,36 @@
         cip_content:
           parameter_defaults:
             ContainerImagePrepare: "{{ cip_config }}"
+            ContainerImageRegistryLogin: true
+
+    # Red Hat Service Accounts have fixed usernames that incorporate a pipe character
+    # there is a lot of work required to handle this vs. a straight username because of to_nice_yaml
+    # when the rhcs5 portion of the build is started.
+    #
+    # Therefore it is easier to use the config.json method for podman.
+    - name: Setup Docker registry authentication
```

Comment:
I'm currently investigating this too. Not saying I reject your solution here but I'm looking if we can do easier.